src/views/SearchResults/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { Box, SimpleGrid, Flex, GridItem, Tag, Text } from "@chakra-ui/react";
+import { useEffect, useMemo } from "react";
+import { Link } from "react-router-dom";
+import { fetchPackages } from "../../api/package/packages";
+import { useQueryParams } from "../../hooks/useQueryParams";
+import { useRequest } from "../../hooks/useRequest";
+
+export function SearchResults() {
+  const query = useQueryParams().get("q") ?? "";
+  const [requestPackages, { data }] = useRequest(fetchPackages);
+
+  useEffect(() => {
+    void requestPackages();
+  }, [requestPackages]);
+
+  const results = useMemo(() => {
+    return data?.packages.filter((item) =>
+      JSON.stringify(item).includes(query)
+    );
+  }, [query, data]);
+
+  return (
+    <SimpleGrid columns={[1, null, 3, null, 5]} p={6} spacing={6}>
+      {results &&
+        results.map((pkg) => {
+          return (
+            <Link key={pkg.name} to={`/packages/${pkg.name}/v/${pkg.version}`}>
+              <GridItem
```

Comment:
Another idea for another time - we'll probably want to split this markup into two separate components: `<PackageList />` and `<PackagePreview />`, that way we can storybook & test them separately. I imagine the component api would be as simple as `<PackageList packages={results} />